app/assets/stylesheets/components/_header.scss

```diff
@@ -15,6 +15,14 @@
     }
   }
 
+  @include govuk-media-query($from: tablet, $until: desktop) {
+    .govuk-header__container {
+      .govuk-header__navigation-list {
+        margin-top: 1.5rem;
+      }
+    }
+  }
+
```

Comment:
```suggestion
```

and then change

https://github.com/DFE-Digital/itt-mentor-services/blob/37dac44f58f15a9efa719877dbf5d9b69268525c/app/assets/stylesheets/components/_header.scss#L2

to

```scss
  @include govuk-media-query($from: desktop) {
```